tools/Microsoft.Health.Dicom.SchemaManager/Properties/Resources.resx

```diff
@@ -0,0 +1,155 @@
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ApplyCommandDescription" xml:space="preserve">
+    <value>Applies the specified SQL schema update to the supplied connection string for the given server.</value>
+  </data>
+  <data name="AuthenticationTypeDescription" xml:space="preserve">
+    <value>The authentication type to connect with.</value>
+    <comment>Valid values are ManagedIdentity and ConnectionString.</comment>
+  </data>
+  <data name="ConnectionStringOptionDescription" xml:space="preserve">
+    <value>The connection string of the SQL server to apply the schema update.</value>
+  </data>
+  <data name="ConnectionStringRequiredValidation" xml:space="preserve">
+    <value>You must include a connection string.</value>
+  </data>
+  <data name="ForceOptionDescription" xml:space="preserve">
+    <value>The schema migration is run without validating the schema verison.</value>
+  </data>
+  <data name="IncompatibleVersionFormatString" xml:space="preserve">
+    <value>The specified schema version "{0}" is incompatible.</value>
+    <comment>{0} is a version.</comment>
+  </data>
+  <data name="LatestOptionDescription" xml:space="preserve">
+    <value>The versions between current version and latest available version to apply.</value>
```

Comment:
Maybe something like "Applies versions between the current version and the latest available"?